src/Symfony/Component/Cache/Adapter/TagInvalidationInterface.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache\Adapter;
+
+use Psr\Cache\CacheItemPoolInterface;
+use Psr\Cache\InvalidArgumentException;
+
+/**
+ * Interface for invalidating cached items using tag hierarchies.
+ *
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+interface TagInvalidationInterface extends CacheItemPoolInterface
+{
+    /**
+     * Invalidates cached items using tag hierarchies.
+     *
+     * @param string $tag A tag hierarchy to invalidate.
+     *
+     * @return bool True on success.
+     *
+     * @throws InvalidArgumentException When $tag is not valid.
+     */
+    public function invalidate($tag);
```

Comment:
Or we could allow string|array (would be my personal preference).
Same for TaggedCacheItemInterface::tag(), isn't it?